documents/operating.md

```diff
@@ -11,6 +12,17 @@ Contents:
 6.  Policies
 
 ---------------------------------------------------------------------
+SECTION 0: 📓📓📓 Definitions 📓📓📓
+---------------------------------------------------------------------
+
+* **Bad Idea Factory Limited Liability Company ("the Company")** is a Limited Liability Company domiciled in Pennsylvania.
+* The Company may also be referred to as "**Bad Idea Factory**" or "**BIFFUD**".
+* "**Corporate Overlord**" shall mean an "owner" of the Company under Pennsylvania state law.
+* The "**Board of Directors**" ("the **Board**") of the Company, as defined by Pennsylvania state law, shall consist of all current Corporate Overlords of the Company.
+* "**Consensus**" shall mean TKTKTK.
+* "**Quorum**" shall mean TKTK
```

Comment:
Right, so the point of a Quorum for me is to ensure that decisions being made are in fact representative of the org, irrespective of whether it's in person or asynchronous.  I want to avoid big decisions being made by a minority of the org, basically.

Each decision should be some % of votes to pass, with some % of the org weighing in.

It may be the case that we believe there to be different classes of decision having different values for those thresholds.

Also, we can just link/borrow if something's in the wrong pull request.